tema1sapt2/tema1sapt2/Program.cs

```diff
@@ -0,0 +1,216 @@
+﻿//Exercise 1<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+Console.Write("Introduceti primul numar" + "  ");
+int num1 = Convert.ToInt32(Console.ReadLine());
+Console.Write("Introduceti al doilea numar" + "  ");
+int num2 = Convert.ToInt32(Console.ReadLine());
+Console.Write("Introduceti al treilea numar" + "  ");
+int num3 = Convert.ToInt32(Console.ReadLine());
+if ((num1 == num2) & (num2 == num3) & (num1 == num3))
+{
+    Console.WriteLine("Cele 3 numere sunt egale");
+}
+else if (num1 == num2)
+{
+    Console.WriteLine("Numarul 1 si numarul 2 sunt egale");
+}
+else if (num2 == num3)
+{
+    Console.WriteLine("Numarul 2 si numarul 3 sunt egale");
+}
+else if (num1 == num3)
+{
+    Console.WriteLine("Numarul 1 si numarul 3 sunt egale");
+}
+if ((num1 > num3) & (num1 > num2))
+{
+    Console.WriteLine("Maximum number is " + "  " + num1);
+}
+
+else if (num2 > num3)
+{
+    Console.WriteLine("Maximum number is" + "  " + num2);
+}
+else
+{
+    Console.WriteLine("Maximum number is" + "  " + num3);
+}
+Console.WriteLine("       ");
+
+if ((num1 < num3) & (num1 < num2))
+{
+    Console.WriteLine("Minimum number is " + "  " + num1);
+}
+
+
+else if (num2 < num3)
+{
+    Console.WriteLine("Minimum number is" + "  " + num2);
+}
+else
+{
+    Console.WriteLine("Minimum numer is" + "  " + num3);
+}
+
+//Exercise 2<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>
+
+int max = num1;
+max = (num1 > num2) ? ((num1 > num3) ? num1 : num3) : ((num2 > num3) ? num2 : num3);
+Console.WriteLine("biggest number among " + num1 + ", " +
+                   num2 + " and " + num3 + " is: " + max);
+int min = num1;
+min = (num1 < num2) ? ((num1 < num3) ? num1 : num3) : ((num2 < num3) ? num2 : num3);
+Console.WriteLine("Smallest number among " + num1 + ", " +
+                   num2 + " and " + num3 + " is: " + min);
+
+//Exercise 3<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>
+Console.WriteLine("Introduceti numarul de zile lucratoare pana la urmatoarea marire de salariu ");
+int days = Convert.ToInt32(Console.ReadLine());
+int years = days / 365;
+days = days - (years * 365);
+int months = days / 30;
+days = days - (months * 30);
+if (years == 1)
+
+{
+    Console.Write(years + "year");
```

Comment:
"year " - add a space after the word so that the console output can be easily read.